.dockerignore

```diff
@@ -0,0 +1,8 @@
+.git
+.gitignore
+Jenkinsfile
+README.md
+docs
+log
+spec
+tmp
```

Comment:
Are you sure these directories are not needed - have you tested your changes against E2E checks - are your sure spec folder is not needed part of the build.

Reply:
I took it originally from publisher and modify the list accordingly: https://github.com/alphagov/publisher/blob/main/.dockerignore